admin/views/tabs/network/general.php

```diff
@@ -13,6 +13,14 @@
 	exit();
 }
 
+echo '<div class="tab-block">';
+printf(
+	/* translators: 1: h2 open tag; 2: h2 close tag. */
+	esc_html__( '%1$sGeneral%2$s', 'wordpress-seo' ),
```

Comment:
I don't think we should put HTML within translatable strings. A best practice is to _move out_ HTML from strings. Also, in this specific case, translators might mess with the headings hierarchy, which is not good 🙂